sdk/metric/controller/push/push.go

```diff
@@ -73,24 +74,26 @@ var _ Ticker = realTicker{}
 // using the provided integrator, exporter, collection period, and SDK
 // configuration options to configure an SDK with periodic collection.
 // The integrator itself is configured with the aggregation selector policy.
-func New(integrator export.Integrator, exporter export.Exporter, period time.Duration, opts ...Option) *Controller {
+func New(integrator export.Integrator, exporter export.Exporter, configLoaderCh chan struct{},
+	period time.Duration, opts ...Option) *Controller {
 	c := &Config{ErrorHandler: sdk.DefaultErrorHandler}
 	for _, opt := range opts {
 		opt.Apply(c)
 	}
 
 	impl := sdk.NewAccumulator(integrator, sdk.WithErrorHandler(c.ErrorHandler))
 	return &Controller{
```

Comment:
We can implement a ConfigWatcher, register it with the notifier, and pass it to controller.

Reply:
Let me know if I implemented this in the way you were thinking.